File: scripts/reset-v71.mjs
import { execSync } from 'child_process';

const V71_COMMIT = 'ebde277df8a81dcf73ba7348a054b2ccaf86296c';

try {
  console.log('Resetting to V71...');
  execSync(`git reset --hard ${V71_COMMIT}`, { 
    cwd: '/vercel/share/v0-project',
    stdio: 'inherit' 
  });
  console.log('Successfully reset to V71');
} catch (error) {
  console.error('Error:', error.message);
}

File: scripts/reset-v71.py
#!/usr/bin/env python3
import subprocess
import sys
import os

os.chdir('/vercel/share/v0-project')

# V71のコミットハッシュに戻す
result = subprocess.run(['git', 'reset', '--hard', 'ebde277df8a81dcf73ba7348a054b2ccaf86296c'], 
                       capture_output=True, text=True)

print(result.stdout)
if result.stderr:
    print(result.stderr, file=sys.stderr)

sys.exit(result.returncode)
